tests/HUBActionRegistryTests.m

```diff
@@ -58,7 +58,7 @@ - (void)testRegisteringFactoryAndCreatingAction
     HUBActionFactoryMock * const factory = [[HUBActionFactoryMock alloc] initWithActions:@{actionIdentifier.namePart: action}];
     [self.actionRegistry registerActionFactory:factory forNamespace:actionIdentifier.namespacePart];
     
-    XCTAssertEqual([self.actionRegistry createCustomActionForIdentifier:actionIdentifier], action);
+    XCTAssertEqualObjects([self.actionRegistry createCustomActionForIdentifier:actionIdentifier], action);
```

Comment:
I wonder if we should use `XCTAssertTrue([self.actionRegistry createCustomActionForIdentifier:actionIdentifier] == action)` for that then, just to be explicit about it.